test/snowplow_test.dart

```diff
@@ -12,6 +12,7 @@
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:snowplow_tracker/configurations/gdpr_configuration.dart';
+import 'package:snowplow_tracker/configurations/network_configuration.dart';
```

Comment:
Nit: this import is not necessary.

Reply:
It is now that the correct version of the file is here